scripts/antlr4/Cypher.g4

```diff
@@ -954,10 +963,6 @@ kU_NonReservedKeywords
         | TYPE
         | USE
         | WRITE
-        | SINGLE
```

Comment:
Why are these being removed?

Reply:
These should be related to Sterling's commit above